mountpoint-s3-client/src/object_client.rs

```diff
@@ -267,6 +269,12 @@ impl PutObjectParams {
         self.trailing_checksums = value;
         self
     }
+
+    /// Set the storage class.
+    pub fn storage_class(mut self, value: Option<String>) -> Self {
```

Comment:
nit: the idiomatic way to do builders like this is that it should take just `String` as input, and the callers only call it if they want to set a non-None value